Bottom: 83d8fbb4dc03c0c27085c450d7033ab13a55a803
Top:    ff9c876e09850207b9469a2a8bf8384edfed5d54
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-10-05 23:09:43 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
